tensorbay/client/dataset.py

```diff
@@ -177,20 +177,6 @@ def list_draft_titles_and_numbers(
         """
         yield from self._list_drafts(start=start, stop=stop)
 
```

Comment:
I think you comment the function due to the word "hide"
As the function is actually deleted maybe you can use "remove the function temporarily" as your commit message